sharding/p2p/topics.go

```diff
@@ -0,0 +1,37 @@
+package p2p
+
+import (
+	"reflect"
+
+	pb "github.com/prysmaticlabs/geth-sharding/sharding/p2p/proto/v1"
+)
+
+// Mapping of message topic enums to protobuf types.
+var topicTypeMapping = map[pb.Topic]reflect.Type{
+	pb.Topic_COLLATION_BODY_REQUEST:  reflect.TypeOf(pb.CollationBodyRequest{}),
```

Comment:
Is the naming convention for proto messages in Go typically like this? Is there any way it could become camel case or nah?

Reply:
This is generated code, nothing that can be done about it.